deepmd/pt/utils/auto_batch_size.py

```diff
@@ -12,6 +12,32 @@
 
 
 class AutoBatchSize(AutoBatchSizeBase):
+    """Auto batch size.
+
+    Parameters
+    ----------
+    initial_batch_size : int, default: 1024
+        initial batch size (number of total atoms) when DP_INFER_BATCH_SIZE
+        is not set
+    factor : float, default: 2.
+        increased factor
+    returned_dict:
+        if the batched method returns a dict of arrays.
```

Comment:
bool, default: false

Reply:
the argument is removed according to @njzjz 's suggestion.